sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.SasVersion.cs

```diff
@@ -20,7 +20,8 @@ public void SetSasVersion()
                 // WARN! Never ever override SAS version in Live mode. We should test the default there.
 
                 // TODO Uncomment this to record/playback with different sas version
-                // SetSasVersion("2020-08-04");
+                // TODO https://github.com/Azure/azure-sdk-for-net/issues/19575
+                SetSasVersion("2020-08-04");
```

Comment:
You could try 
`SetSasVersion(StorageVersionExtensions.MaxVersion.ToString().Replace("V","").Replace("_", "-"));` so that we don't have to revisit this often.
The reason for this logic is that we shouldn't use `ToVersionString` for Queues in this case. But we can derive this from V2020_08_04.